tests/framework/framework.go

```diff
@@ -132,14 +132,14 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
 	// clients
 	kcs, err := f.GetKubeClient()
 	if err != nil {
-		err = errors.Wrap(err, "ERROR, unable to create K8SClient")
+		return nil, errors.Wrap(err, "ERROR, unable to create K8SClient")
```

Comment:
don't need else since you've returned above.

Reply:
You are right, removing else.